ChangeLog.rst

```diff
@@ -13,6 +13,10 @@ upgrading a MetalK8s 0.1 cluster to MetalK8s 0.2 (:ghissue:`147`)
 storage related actions. Please see :ref:`upgrade_from_MetalK8s_before_0.2.0`
 (:ghissue:`153`)
 
+Bugs fixed
```

Comment:
Not really a bug...

Reply:
I wondered a lond time about this... So it's a feature added with a `ghissue` ?

Reply:
There are only 'Issues' on Github (and 'Notes' in a 'Project'), so yeah. It's a bit of a misnomer, but we've been using 'Issues' to track features for quite a while already...